### DISCUSSION TITLE
Could we use wireless to replace link cable?

### DISCUSSION THREAD

**Discussion by PigletTubeYT** [profi200/open_agb_firm#100] [General]
Pretty much like the Nintendo DS Wireless

**Comment by profi200** [comment]
Short answer: Not really.

Long:
The link cable protocol has very tight timing requirements. WiFi has way too high latency to make it work reliably. The only way i can see this ever working is by heavily patching the game to be more forgiving about timings. The wireless adapter on the other hand may work but it still requires lots of patching.
